src/Donation/Donation.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Give\Donation;
+
+use function Give\Helpers\Form\Template\Utils\Frontend\getPaymentId;
+
+/**
+ * Class Donation
+ *
+ * @package Give\Donation
+ */
+class Donation {
+	private $id;
+
+	/**
+	 * Donation constructor.
+	 *
+	 * @param int|null $id
+	 */
+	public function __construct( $id = null ) {
+		$this->id = $id ?: getPaymentId();
+	}
+
```

Comment:
Remove extra line break

Reply:
This is used code or you can say incomplete Donation Model which we are not using.